### ISSUE TITLE
Wrong information in the tray icon when plugged in

### ISSUE THREAD

**Issue by Alian51** [gawindx/WinNUT-Client#45]
With mouse over the tray icon, the mention On Battery (xx%) is mentioned even if the UPS is on AC (at least in the French version, maybe only a translation mismatch).
The tray icon is well reacting when plugging / unplugging the UPS on / from AC, becoming green or red, but the indication still showing On Battery.
BTW, very good app!
[Tray icon mouse over.docx](https://github.com/gawindx/WinNUT-Client/files/5507228/Tray.icon.mouse.over.docx)

**Comment by gawindx** [comment]
this is not a bug in the sense that, even if your ups is plugged into the mains, your battery is not necessarily 100% charged (which is the case on your capture), especially after a power cut .

In addition, the fact of displaying the state of charge also makes it possible to see a battery problem coming, such as a battery which no longer takes the charge correctly or which does not maintain its nominal voltage (and therefore its charge).

**Comment by gawindx** [comment]
I read a little quickly (and badly because, like you, I am French).
indeed, this is not normal.
that the battery level is always displayed is normal, but that we are all the time on battery even connected to the mains is not.

has to be corrected for the next version (and it is indeed possible that this is a translation error).

**Comment by Alian51** [comment]
Thanks for your feedback ! And looking forward to installing the new version as soon as released!

**Comment by gawindx** [comment]
I come back to you concerning this problem (I am putting my hands in the code so I am taking advantage of it).

It seems that your problem is not due to a translation problem but rather to a bad treatment (of my fault therefore) of the return value of the state of your ups, value which seems to be returned with an unnecessary space (thus fault of your inverter or NUT server).

What is stupid is that I had bypassed the problem for the value of the icon but I did not think of doing it for the text ...

this will be fixed in v2.0.5 (should be released soon).

**Comment by Alian51** [comment]
Great!

Thanks for the follow-up.

 

Enjoy your week.

 

 

De : Decaux Nicolas <notifications@github.com> 
Envoyé : mardi 12 janvier 2021 19:57
À : gawindx/WinNUT-Client <WinNUT-Client@noreply.github.com>
Cc : Alian51 <avandeneede@free.fr>; Author <author@noreply.github.com>
Objet : Re: [gawindx/WinNUT-Client] Wrong information in the tray icon when plugged in (#45)

 

I come back to you concerning this problem (I am putting my hands in the code so I am taking advantage of it).

It seems that your problem is not due to a translation problem but rather to a bad treatment (of my fault therefore) of the return value of the state of your ups, value which seems to be returned with an unnecessary space (thus fault of your inverter or NUT server).

What is stupid is that I had bypassed the problem for the value of the icon but I did not think of doing it for the text ...

this will be fixed in v2.0.5 (should be released soon).
